src/lib.rs

```diff
@@ -195,6 +195,13 @@ impl Certificate {
         let cert = try!(imp::Certificate::from_der(der));
         Ok(Certificate(cert))
     }
+    /// Parses a PEM-formatted X509 certificate.
+    /// If the PEM file contains more than one certificate the last one is used
+    /// and the others are ignored.
```

Comment:
OpenSSL will take the first IIRC. Not sure what Windows will do - I think fail?

Reply:
Your call - I am rather n00b wrt all this - do you want me to change it to take the first?

Reply:
For now let's just leave the behavior unspecified and see if it causes issues. It's pretty broken to depend on the behavior with a multi-cert file anyway.